SPIDisplay/CS/README.md

```diff
@@ -0,0 +1,454 @@
+---

```

Comment:
Can you make sure all images in this ReadMe populate?

Reply:
Updated the image paths and images are coming up in readme.